.changeset/brave-impalas-switch.md

```diff
@@ -0,0 +1,24 @@
+---
+'example-backend': patch
```

Comment:
No need for changesets for the example backend or any other package that isn't published